vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -67,6 +67,9 @@ module IntegersAndNumbers =
     // The next line prints a list that includes tuples, using '%A' for generic printing.
     printfn "The table of squares from 0 to 99 is:\n%A" sampleTableOfSquares
 
+    // This is a sample integer with a type annotation
```

Comment:
@CameronAavik 

Don't worry about squashing, I will squash when I pull the PR.

do you think that the sample with annotation should be closer to the sample integer without annotation.  It seems that would that make it more relevant?